asistente_ladm_col/utils/qgis_utils.py

```diff
@@ -895,6 +895,32 @@ def copy_csv_to_db(self, csv_layer, db, target_layer_name):
 
         return True
 
+    def remove_all_layers_and_groups(self):
+        # Remove layers
+        QgsProject.instance().removeAllMapLayers()
+
+        # Remove groups
+        root = QgsProject.instance().layerTreeRoot()
+        for group in [child for child in root.children() if child.nodeType() == 0]:
+            root.removeChildNode(group)
+
+        self.map_refresh_requested.emit()
+
+    def get_all_layers_on_map(self):
```

Comment:
Not needed, just use `list(QgsProject.instance().mapLayers().values())`

Reply:
It's done, thanks.